packages/cli/src/cmds/preview/preview.ts

```diff
@@ -86,7 +86,7 @@ export default class Preview extends YargsCommand {
       string: true,
     });
     yargs.option("env", {
-      description: "select an existed env for the project",
+      description: "select an existing env for the project",
```

Comment:
Upper case for the first character to keep consistent?

Reply:
`env` -> `environment`?

Reply:
Fixed.